back-end/api/quickstart/helper_test.py

```diff
@@ -33,8 +33,7 @@ def get_test_like_comment_fields(comment_id=1):
 
 def get_test_comment_fields(i=1, postid="testpostid"):
```

Comment:
postid not used anymore so you can take it out

Reply:
Done